src/ngx-image-zoom.component.html

```diff
@@ -1,7 +1,7 @@
 <div #zoomContainer class="ngxImageZoomContainer"
      [style.width.px]="this.thumbWidth" [style.height.px]="this.thumbHeight">
 
-    <img #imageThumbnail class="ngxImageZoomThumbnail" src="{{ thumbImage }}" width="100%" height="100%"
+    <img #imageThumbnail class="ngxImageZoomThumbnail" [src]="thumbImag" width="100%" height="100%"
```

Comment:
Typo in variable name
```suggestion
    <img #imageThumbnail class="ngxImageZoomThumbnail" [src]="thumbImage" width="100%" height="100%"
```